packages/l/libgpiod/xmake.lua

```diff
@@ -0,0 +1,24 @@
+package("libgpiod")
+    set_homepage("https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/about/")
+    set_description("libgpiod - C library and tools for interacting with the linux GPIO character device (gpiod stands for GPIO device)")
+    
+    add_urls("https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/snapshot/libgpiod-$(version).tar.gz",
+             "https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git")
+    add_versions("v2.0.1", "62071ac22872d9b936408e4a067d15edcdd61dce864ace8725eacdaefe23b898")
+    add_versions("v2.0", "1e4dbbe8b3f32adf8818f1b0dfd322876aa9477c")
```

Comment:
why not use sha256?

Reply:
v2.0.1 is a sha256 retrieved with `shasum -a 256 libgpiod-2.0.1.tar.gz`

Reply:
but v2.0 (1e4dbbe8b3f32adf8818f1b0dfd322876aa9477c) is not sha256 

```
shasum -a 256 ~/Downloads/libgpiod-v2.0.tar.gz
a0f835c4ca4a2a3ca021090b574235ba58bb9fd612d8a6051fb1350054e04fdd  /Users/ruki/Downloads/libgpiod-v2.0.tar.gz
```

Reply:
Does it matter? the sha256 one doesn't even validate, why would I put effort into the other 2 if the first one doesn't actually verify?